### DISCUSSION TITLE
Route testing

### DISCUSSION THREAD

**Discussion by Torqueitup** [KillBait/AutoDrive_Course_Editor#80] [Ideas]
Having gone through the arduous task of building a AD network from the ground up, then proceeding to test the viability of the created nodes/links/destinations I began to think about how this testing phase could be better achieved through the use of AD course editor.  It would be nice to enter the start point, then the destination, and have the available routes blink/flash/big red line up to the point where the route is viable.  Breaks or no go's would be quite obvious and could be fixed easily and quickly.  Just a thought.

**Comment by KillBait** [comment]
Sorry, i missed this completely, it's strange, i was just thinking the same thing a few days ago and added it to the wishlist. I'm currently re-doing the whole UI as it's been long overdue, my bad drawn icons and not very efficient UI layout have really annoyed me since day 1. :D.

The next version will be V2.0 as it will look a fair bit different ( but familiar ) and i got a lot of stuff to work on and add, when it will be released, i don't know, i have limit time to work on it, so 'when it done' is all i can give.

Sneak peek..
![Top](https://github.com/user-attachments/assets/24cf164e-3c2e-424c-acdf-ee06f26b2c82)

**Reply by Torqueitup** [reply]
I appreciate all you've done and thanks for the tool.  Brilliant.
Good luck on the new version, I'm looking forward to seeing the new innovations along with FS25
